frontend/src/components/smart/MerchandiseCart.vue

```diff
@@ -0,0 +1,210 @@
+      <template #modal-footer>
+        <div class="cart-footer">
+          <div v-if="cartEntries.length !== 0">
+            <span>{{ $t('market.merchandise.total') }}: {{ totalPrice.toFixed(2) }} {{
+                cartEntries[0].variant.currency
+              }} </span>/
+            <span><CurrencyConverter :skill="fromWeiEther(totalPriceInSkill)" :show-value-in-skill-only="true"/></span>
+          </div>
+          <b-button :disabled="isPlaceOrderButtonDisabled()" variant="primary" @click="openAddressModal">
+            {{ $t('market.merchandise.placeOrder') }}
+          </b-button>
+        </div>
+      </template>
+    </b-modal>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import {BModal} from 'bootstrap-vue';
+import {PropType} from 'vue/types/options';
+import {CartEntry, FileType} from '@/components/smart/VariantChoiceModal.vue';
+import {mapActions} from 'vuex';
+import {fromWeiEther, toBN} from '@/utils/common';
+import CurrencyConverter from '@/components/CurrencyConverter.vue';
+
+interface StoreMappedActions {
+  currentSkillPrice(): Promise<string>;
+}
+
+interface Data {
+  totalPrice: number;
+  totalPriceInSkill: number;
+  skillPrice: number;
+  isOrderLoading: boolean;
+}
+
+export default Vue.extend({
+  data() {
+    return {
+      totalPrice: 0,
+      totalPriceInSkill: 0,
+      skillPrice: 0,
+      isOrderLoading: false,
+    } as Data;
+  },
+
+  props: {
+    cartEntries: {
+      type: Array as PropType<CartEntry[]>,
+      required: true
+    }
+  },
+
+  components: {CurrencyConverter},
+
+  methods: {
+    ...mapActions(['currentSkillPrice']) as StoreMappedActions,
+    fromWeiEther,
+    toBN,
+    async openAddressModal() {
+      this.$root.$emit('merchandise-address-modal', this.totalPriceInSkill, this.cartEntries);
+    },
+    async openCartModal() {
+      this.$root.$emit('merchandise-cart-modal', true);
+    },
+    isFileTypePreview(file: File) {
+      return file.type === FileType.PREVIEW;
+    },
+    subtractQuantity(cartEntry: CartEntry) {
+      cartEntry.quantity--;
+      this.calculateTotalPrice();
+    },
+    addQuantity(cartEntry: CartEntry) {
+      cartEntry.quantity++;
+      this.calculateTotalPrice();
+    },
+    calculateTotalPrice() {
+      this.totalPrice = this.cartEntries.map(cartEntry => +cartEntry.variant.retail_price * cartEntry.quantity)
+        .reduce((a, b) => a + b, 0);
+      this.totalPriceInSkill = this.totalPrice * this.skillPrice;
+    },
+    usdToSkill(cartEntry: CartEntry) {
+      if (!cartEntry?.variant) return;
+      return +cartEntry.variant.retail_price * this.skillPrice;
+    },
+    removeCartEntry(cartEntry: CartEntry) {
+      this.cartEntries.splice(this.cartEntries.indexOf(cartEntry), 1);
+      this.calculateTotalPrice();
+    },
+    isMinusButtonDisabled(cartEntry: CartEntry) {
+      return cartEntry.quantity <= 1;
+    },
+    isPlaceOrderButtonDisabled() {
+      return this.cartEntries.length === 0;
```

Comment:
Might be worth to check for sufficient SKILL balance too.

Reply:
![image](https://user-images.githubusercontent.com/34208222/144885813-a6db3ec9-56a6-4591-90a9-12dd04dc50d2.png)

Reply:
Added it, also a chef kiss - tooltip explaining that the button is disabled because of that